src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -595,3 +616,48 @@ def get_files_path(data_path):
                 break
 
     return files_path
+
+
+def _make_symlink(path, source_path, name, index):
+    base_name = os.path.join(source_path, name + '_' + str(index))
+    logging.info('creating symlink between Path {} and destination {}'.format(source_path, base_name))
```

Comment:
If the minimum Python version on the container is Python 3.6, let's use f-strings here instead: https://realpython.com/python-f-strings/ Same in other places.

Reply:
updated at one other place that was concatenating a string

Reply:
Looks like this still needs to be updated here.